src/presign/participant.rs

```diff
@@ -1108,15 +1108,15 @@ impl PresignKeyShareAndInfo {
         for (other_id, round_three_input) in other_participant_inputs {
             let psi_double_prime = PiLogProof::prove(
                 rng,
-                &PiLogInput::new(
-                    &round_three_input.auxinfo_public.params,
+                &CommonInput::new(
```

Comment:
Ah I see. I mostly didn't like the duplicate naming of `pilog::PiLog...`, but I can see where confusion could come in. I guess the importer can do `use pilog::CommonInput as PiLogCommonInput` but then maybe it's not worth the extra hassle. Thoughts?

Reply:
Yeah, I agree that `pilog::PiLog...` is silly. I don't mind inline import strings, like
```rust
let psi = PiLogProof::prove(
    rng,
    &pilog::CommonInput::new(...),
    &pilog::ProverSecret::new(...),
)?;
```

Otherwise, the `use` statement you describe is also fine -- it reduces `PiLog` duplication in the `pilog` module, which seems worthwhile anyway.